source/common/ssl/context_impl.cc

```diff
@@ -422,10 +422,7 @@ ServerContextImpl::ServerContextImpl(ContextManagerImpl& parent, Stats::Scope& s
   bssl::UniquePtr<GENERAL_NAMES> san_names(
       static_cast<GENERAL_NAMES*>(X509_get_ext_d2i(cert, NID_subject_alt_name, nullptr, nullptr)));
   if (san_names != nullptr) {
-    // TODO(ggreenway): Use range-based for loop when newer BoringSSL build is used:
-    //   for (const GENERAL_NAME* san : *san_names) {
-    for (size_t i = 0; i < sk_GENERAL_NAME_num(san_names.get()); i++) {
-      const GENERAL_NAME* san = sk_GENERAL_NAME_value(san_names.get(), i);
+    for (const GENERAL_NAME* san : san_names.get()) {
```

Comment:
Thanks; much nicer this way

Reply:
Sigh, I have to revert that, since new BoringSSL isn't in the build image yet.